src/tasks.py

```diff
@@ -140,6 +157,28 @@ def execute_yara(self, job: Job, files: List[str]) -> None:
                 f"in {scanned_datasets}/{job.total_datasets} ({dataset_percent:.0%}) of datasets.",
             )
 
+    @staticmethod
+    def get_match_context(
+        data: bytes, matches: List[yara.Match]
+    ) -> Dict[str, Dict[str, Dict[str, str]]]:
+        context = {}
+        for yara_match in matches:
+            match_context = {}
+            for string_match in yara_match.strings:
+                expression_key = string_match.instances[0]
+
+                (before, matching, after) = read_bytes_with_context(
+                    data, expression_key.matched_length, expression_key.offset
+                )
+                match_context[expression_key] = {
```

Comment:
I'll fix the style in a second but... `expression_key` is an object. So match_context is not a `Dict[str, ...]`, so context is not `Dict[..., Dict[str, ...]]`, so something's wrong